ckanext/openafrica/blueprint.py

```diff
@@ -9,14 +9,21 @@ def render_path():
 
     return render_path
 
+def robots_txt():
+    '''display robots.txt'''
```

Comment:
```suggestion
    """display robots.txt"
```
Why mixing `'` and `"`?